databricks/README.md

```diff
@@ -1,28 +1,50 @@
-    1. This will become the base init script that is run at cluster start up.
-    1. Example:
+# Databricks Notebooks with MLFlow, RAPIDS, and Hyperopt
+
+This directory contains a sample notebook that walks through the
+optimization of a random forest model using cuML and hyperopt on the
+DataBricks cloud. It also includes init scripts to install RAPIDS
+automatically on your DataBricks cluster.
+
+It has been tested with RAPIDS 0.13 on DataBricks ML Runtime 6.6. (Support for RAPIDS 0.14 is planned for the near future.)
+
+# Running the demo on DataBricks
+
+## Upload RAPIDS 0.13 Init Script to DBFS
+* Copy `src/rapids_install_cuml0.13_cuda10.0_ubuntu16.04.sh` onto your Databricks dbfs file system.
+    * This will become the base init script that is run at cluster start up.
+    * Example:
     ```shell script
     $ dbfs configure
        ... configure your dbfs client for your account ...
     $ dbfs cp src/rapids_install_cuml0.13_cuda10.0_ubuntu16.04.sh dbfs:/databricks/init_scripts/
     ```
    
-### Create a Cluster
-1. Create your cluster
+## Create and Configure a Cluster
+* Create your cluster:
     1. Select a GPU enabled Databricks runtime. Ex: 6.6 ML 
-        1. Currently 'Use your own Docker container' is not available for ML instances.
-    1. Select a GPU enabled worker and driver type
-        1. **Note** Selected GPU must be Pascal generation or greater. p2.X is not supported.
-        1. Recommended: `g4dn.xxxx` (NVIDIA T4) or `p3.xxxx` (NVIDIA V100)
-    1. Select `Advanced` -> `init_scripts`
-        1. For our example set `init_scripts` to `dbfs:/databricks/init_scripts/rapids_install_cuml0.13_cuda10.0_ubuntu16.04.sh'
-    1. Launch your cluster
-        1. At this point, you should have RAPIDS 0.13 installed in the databricks-ml-gpu conda environment, and can import cudf/cuml modules.         
-
-## Databricks Jobs with MLFlow, RAPIDS, and Hyperopt
+        * Currently 'Use your own Docker container' is not available for ML instances.
+    2. Select a GPU enabled worker and driver type
+        * **Note** Selected GPU must be Pascal generation or greater. p2.X is not supported.
+        * Recommended: `g4dn.xxxx` (NVIDIA T4) or `p3.xxxx` (NVIDIA V100) for AWS users
+    3. Select `Advanced` -> `init_scripts`
+        * Add an init scripts with the location `dbfs:/databricks/init_scripts/rapids_install_cuml0.13_cuda10.0_ubuntu16.04.sh'
+* Launch your cluster
+  * At this point, you should have RAPIDS 0.13 installed in the databricks-ml-gpu conda environment, and can import cudf/cuml modules.
+
+![Setting up init script](imgs/init_script_config.png)
+
+## Launching the notebook
+
+1. Upload the `rapids_airline_hyperopt.ipynb` notebook to your workspace.
+2. Uncomment the "data download" cell and configure it to point to a path of your choice for data download. By default, it will use a smaller (200k row) dataset. This executes fast but doesn't demonstrate the full speedups possible with larger datasets.
```

Comment:
Should we specify that we have a 20M version available for testing if interested?

Reply:
Fair point. Adding it.